src/ray/raylet/scheduling/local_resource_manager.cc

```diff
@@ -403,6 +470,10 @@ std::optional<syncer::RaySyncMessage> LocalResourceManager::CreateSyncMessage(
 
   resources_data.set_resources_available_changed(true);
 
+  const auto now = absl::Now();
+  resources_data.set_idle_duration_ms(
```

Comment:
hmm should we use -1 or something else instead? using to indicate resource not idle could be confusing.

Reply:
So it's currently using `0` to indicate not idle. If we use `-1` for not idle, technically what will be `0` ?